views/project/user.erb

```diff
@@ -84,15 +81,7 @@
                 <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:pl-6" scope="row">
                   <%= user[:email] %>
                 </td>
-                <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
-                  <%== render(
-                    "components/form/policy_select",
-                    locals: {
-                      name: "user_policies[#{user[:ubid]}]",
-                      selected: @user_policies[user[:ubid]]
-                    }
-                  ) %>
-                </td>
+                <td></td>
```

Comment:
`Subject Tag` column is empty

Reply:
This is expected.  You cannot modify the subject tags for existing users in this UI, only the subject tag an invited user will be placed in.  This is because existing users can be in multiple subject tags, and the UI doesn't offer a way to handle that correctly.